mkdocs/docs/protocol/onchain-credential-status.md

```diff
@@ -0,0 +1,190 @@
+# OnChain Credential Status
+
+On-chain identity verification requires the ability to check whether a claim has been revoked using a smart contract. To accomplish this, the claim must include information about the smart contract, such as its address, blockchain, and network. This information can be stored in the `CredentialStatus` structure.
```

Comment:
Please substitute "claim" with "credential". 
Also fix wording this way:
To accomplish this, the credential should have `CredentialStatus` structure of type `Iden3OnchainSparseMerkleTreeProof2023` and include revocation nonce with information about the smart contract, such as its address, blockchain, and network.